2025-02-11 18:47:40,830 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\src\\models\\configs\\letter_mapping.json'
2025-02-11 18:47:49,695 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\src\\models\\configs\\letter_mapping.json'
2025-02-11 18:50:15,084 - ERROR - Error loading checkpoint: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:15,085 - ERROR - Error loading model: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:15,086 - ERROR - Error in application: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:24,377 - ERROR - Error loading checkpoint: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:24,378 - ERROR - Error loading model: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:24,378 - ERROR - Error in application: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:51:19,571 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:19,572 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:31,575 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:31,575 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:41,868 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:41,870 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:36,583 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:36,587 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:40,250 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:40,250 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:40,418 - ERROR - Error making prediction: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 18:59:40,420 - ERROR - Error in application: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 19:04:15,124 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:15,124 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:18,533 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:18,533 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:18,742 - ERROR - Error making prediction: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 19:04:18,743 - ERROR - Error in application: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 19:08:46,409 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:08:46,409 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:08:52,534 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:08:52,535 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:43,401 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:43,404 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:48,339 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:48,340 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:07:14,845 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\src\\models\\configs\\letter_mapping.json'
2025-02-13 18:11:17,575 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:11:17,577 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:11:22,027 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:11:22,030 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:36,344 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:36,348 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:46,635 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:46,637 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:26,208 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:26,210 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:30,808 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:30,810 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:17,174 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:17,178 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:20,255 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:20,256 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:54:57,982 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:54:57,983 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:55:01,538 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:55:01,539 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:55:01,620 - ERROR - Error in application: 'ThamudicApp' object has no attribute 'predict'
2025-02-14 16:57:31,419 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:31,421 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:34,774 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:34,775 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:34,903 - ERROR - Error making prediction: 'box_left'
2025-02-14 16:57:34,904 - ERROR - Error in application: 'box_left'
2025-02-14 16:58:34,196 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:58:34,198 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:58:37,935 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:58:37,938 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:03:59,037 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:03:59,038 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:04:02,369 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:04:02,371 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:04:03,339 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:05:54,218 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:54,221 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:57,185 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:57,188 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:58,089 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:06:09,050 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:09,051 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:12,131 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:12,133 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:13,001 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:09:12,761 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:12,762 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:15,643 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:15,645 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:16,541 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:12:31,158 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:31,159 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:34,182 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:34,184 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:35,073 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:13:40,650 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:40,651 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:43,823 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:43,824 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:44,715 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:15:39,617 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:39,620 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:42,611 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:42,612 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:43,498 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:16:47,321 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:47,323 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:50,188 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:50,190 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:51,063 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:17:11,970 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:11,971 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:14,577 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:14,579 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:15,445 - ERROR - Error in application: name 'confidences' is not defined
